wmfdata/spark.py

```diff
@@ -331,10 +343,47 @@ def run(commands, format="pandas", session_type="yarn-regular", extra_settings={
         elif format == "raw":
             overall_result = overall_result.collect()
 
-    # (re)start a timeout on SparkSessions in Yarn after the result is collected.
-    # A SparkSession used by this run function
-    # will timeout after 5 minutes, unless used again.
     if PREDEFINED_SPARK_SESSIONS[session_type]["master"] == "yarn":
-        start_session_timeout(spark_session, 3600)
+        start_session_timeout(spark_session)
 
     return overall_result
+
+def load_parquet(local_path, hive_db, hive_table=None, overwrite=False):
+    """
+    Loads a Parquet file into the Data Lake and registers it as a Hive table.
+
+    Arguments:
+    * `local_path`: the path (relative or absolute) to the file you want to load
```

Comment:
Why support only local paths?  If you use fsspec, this could be any fully qualified URL as long as it has an fsspec implementation.  

Here's an [example](https://gitlab.wikimedia.org/repos/data-engineering/workflow_utils/-/blob/main/workflow_utils/artifact/cache.py#L113-115) of how I'm using it to sync artifacts to hdfs from almost anywhere.  Basically:

```python
with fspec.open(input_url).open as input_stream:
    with fsspec.open(dest, mode='wb').open() as output_stream:
        output_stream.write(input_stream.read())
```

in this case input_url could be anything, and output_url can be your `hdfs:///` (or probably better `arrow_hdfs:///` path. (ATM, fsspec will use a deprecated pyarrow API with just 'hdfs:///' URLs.  Marcel and I just realized this and will figure out how best to use fsspec + pyarrow + hdfs soon).

Anyway, I'm not sure if I am recommending for you to do this, but just wanted you to be aware that it is possible.
Likely you'd only ever have parquet files locally or already in HDFS though, so you could just vary how to read and then put the file (I guess i the file is already in HDFS you don't need to put it there) based on URI proto.